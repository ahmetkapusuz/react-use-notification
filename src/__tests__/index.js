import React from 'react';
import { Notification, useNotification } from '../index';
import renderer from 'react-test-renderer';

test('Should render successfully', () => {
  const component = renderer.create(<Notification />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
