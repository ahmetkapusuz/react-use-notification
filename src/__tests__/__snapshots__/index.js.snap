// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render successfully 1`] = `
<div
  className="Notification__Container-sc-1kgx9f5-0 ekMjxr"
>
  <div
    className="Notification__MessageContainer-sc-1kgx9f5-1 bUxgjx"
  >
    <div
      className="Notification__IconContainer-sc-1kgx9f5-2 kzeCtx"
    >
      <div
        className="Notification__Icon-sc-1kgx9f5-3 joNqKH"
      >
        ✔
      </div>
    </div>
    <div
      className="Notification__TextContainer-sc-1kgx9f5-4 becpoR"
    />
    <div
      className="Notification__CloseButton-sc-1kgx9f5-5 dToYRu"
    >
      ×
    </div>
  </div>
</div>
`;
